### PR TITLE
Make WebSocketSettings.DisablePayloadMasking throw PNSE.

### LINKED ISSUE #537: WebSocketSettings.DisablePayloadMasking not supported
According to @mconnew this property is not supported but currently simply does nothing when used, it needs to throw PNSE and a unit test confirming this should be written at the same time.

### PR DESCRIPTION
**Pull request by shmao**
Fix #537.

### CODE REVIEW

**Comment by shmao** [comment]
@dotnet-bot test outerloop please

File: src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportSettings.cs [hunk 1]
```diff
@@ -21,14 +21,12 @@ public sealed class WebSocketTransportSettings : IEquatable<WebSocketTransportSe
...
         private TimeSpan _keepAliveInterval;
         private string _subProtocol;
-        private bool _disablePayloadMasking;
 
         public WebSocketTransportSettings()
         {
             _transportUsage = WebSocketDefaults.TransportUsage;
             _keepAliveInterval = WebSocketDefaults.DefaultKeepAliveInterval;
             _subProtocol = WebSocketDefaults.SubProtocol;
-            _disablePayloadMasking = WebSocketDefaults.DisablePayloadMasking;
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportSettings.cs]: We should remove this too then? 

https://github.com/dotnet/wcf/blob/master/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/TransportDefaults.cs#L324

**Comment by iamjasonp** [comment]
LGTM

File: src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportSettings.cs (hunk 1 above)
Comment (shmao) [inline src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportSettings.cs]: You're right. Removed it.

**Pull request merged**